testdata/page_get.json

```diff
@@ -13,75 +13,81 @@
     "Tags": {
       "id": ";s|V",
       "type": "multi_select",
-      "multi_select": [
```

Comment:
Hmm, seems like my formatter automatically changed this up. Not sure if this matters to you, let me know if you would like me to revert this.